src/main/java/com/salesforce/dataloader/mapping/LoadMapper.java

```diff
@@ -73,7 +76,7 @@ protected void putPropertyEntry(Entry<Object, Object> entry) {
     public Row mapData(Row localRow) {
         Row mappedData = new Row();
         for (Map.Entry<String, Object> entry : localRow.entrySet()) {
-            String sfdcName = getMapping(entry.getKey());
+            String sfdcName = getMapping(entry.getKey(), true);
```

Comment:
This was used by partner api to map from csv column name to sdfc name, we need to exact mapping to process this as well.